e2e/aws/rds_test.go

```diff
@@ -368,3 +451,264 @@ func waitForDatabaseServers(t *testing.T, auth *service.TeleportProcess, wantNam
 		}
 	}, 1*time.Minute, time.Second, "waiting for the database service to heartbeat the databases")
 }
+
+// rdsAdminInfo contains common info needed to connect as an RDS admin user via
+// password auth.
+type rdsAdminInfo struct {
+	endpoint,
+	username,
+	password string
+}
+
+func connectAsRDSPostgresAdmin(t *testing.T, ctx context.Context, instanceID string) *pgx.Conn {
+	t.Helper()
+	info := getRDSAdminInfo(t, ctx, instanceID)
+	pgCfg, err := pgx.ParseConfig(fmt.Sprintf("postgres://%s/?sslmode=require", info.endpoint))
```

Comment:
hmm. so the `require` mode requires SSL connection but it won't verify (as CA is not provided)? maybe leave a comment.

Reply:
oh, I had this while I was just getting everything working.
I'll update this to download the AWS RDS CA and use that to verify.

Reply:
I think `require` is fine, the database service will download it anyway.

Reply:
I went ahead and downloaded the global cert bundle for the tests. Only 180kb size, and we are sending the master user password for these test databases.